ckanext/xloader/tests/test_loader.py

```diff
@@ -987,23 +987,6 @@ def test_simple_large_file(self, Session):
             u"text",
         ]
 
-    def test_simple_large_file(self, Session):
```

Comment:
what was this dropped?

Reply:
It's a duplicate of the test function immediately above it.